src/gateway/get-started/proxy-caching.md

```diff
@@ -1,73 +1,190 @@
+   `application/json; charset=utf-8`. `cache_ttl` instructed the plugin to flush values after 30 seconds.
+   
+   The final option `config.strategy=memory` specifies the backing data store for cached responses. More
+   information on `strategy` can be found in the the [parameter reference](/hub/kong-inc/proxy-cache/) 
+   for the Proxy Cache plugin.
+
+1. **Validate**
+
+   You can check that the Proxy Cache plugin is working by sending `GET` requests and examining
+   the returned headers. In step two of this guide, [services and routes](/gateway/latest/get-started/services-and-routes),
+   you setup a `/mock` route and service that can help you see proxy caching in action.
+
+   First, make an initial request to the `/mock` route. The Proxy Cache plugin responds status 
+   information headers prefixed with `X-Cache`, so use `grep` to filter for that information:
+
+   ```
+   curl -i -s -XGET http://localhost:8000/mock/requests | grep X-Cache
+   ```
+  
+   On the initial request, there should be no cached responses, and the headers will indicate this with
+   `X-Cache-Status: Miss`.
+
+   ```
+   X-Cache-Key: c9e1d4c8e5fd8209a5969eb3b0e85bc6
+   X-Cache-Status: Miss
+   ``` 
+
+   Within 30 seconds of the initial request, repeat the command to send an identical request and the
+   headers will indicate a cache `Hit`:
+
+   ```
+   X-Cache-Key: c9e1d4c8e5fd8209a5969eb3b0e85bc6
+   X-Cache-Status: Hit
+   ```
+
+   The `X-Cache-Status` headers can return the following cache results:
+   
+   |State| Description|
+   |---|---|
+   |Miss| The request could be satisfied in cache, but an entry for the resource was not found in cache, and the request was proxied upstream.|
+   |Hit| The request could be satisfied in cache, but an entry for the resource was not found in cache, and the request was proxied upstream.|
+   |Refresh| The resource was found in cache, but could not satisfy the request, due to Cache-Control behaviors or reaching its hard-coded `cache_ttl` threshold.|
+   |Bypass| The request could not be satisfied from cache based on plugin configuration.|
+
+### Service level proxy caching
+
+The Proxy Cache plugin can be enabled for specific services. The request is the same as above, but posted to the service URL:
+
+```sh
+curl -X POST http://localhost:8001/services/example_service/plugins \
+   --data "name=proxy-cache" \
+   --data "config.request_method=GET" \
+   --data "config.response_code=200" \
+   --data "config.content_type=application/json; charset=utf-8" \
+   --data "config.cache_ttl=30" \
+   --data "config.strategy=memory"
 ```
 
-Depending on your configuration, the response header will be composed of many different fields. Notice the integer values in the following fields:
+### Route level proxy caching
 
-* `X-Kong-Proxy-Latency`
+The Rate Limiting plugin can be enabled for specific routes. The request is the same as above, but posted to the route URL:
```

Comment:
```suggestion
The Proxy Caching plugin can be enabled for specific routes. The request is the same as above, but the request is sent to the route URL:
```

We haven't had time to codify this as a rule but we are moving towards the google style guide for cases like this. 

You could also rewrite this by saying '... but the `POST` request is sent to the route URL'